Question 2 Main

```diff
@@ -0,0 +1,41 @@
+package week.pkg5.objects.q2;
+
+import java.util.Scanner;
+
+/**
+ *
+ * @author Dylan Tkach
+ */
+public class Week5ObjectsQ2 {
+
+    /**
+     * @param args the command line arguments
+     */
+    public static void main(String[] args) {
+        Scanner keyboard = new Scanner(System.in);
+        
+        int speed;
+        int yearModel;
+        String make;
+        
+        System.out.println("Please enter the model year of the car now: ");
+        yearModel = keyboard.nextInt();
+        System.out.println("Now, enter the make of the car: ");
+        make = keyboard.next();
+        
+        Car vehicle = new Car(yearModel, make);
+        
+        for(int x = 1; x <= 5; x++){
+            vehicle.accelerate();
+            speed = vehicle.getSpeed();
+            System.out.println("The speed of the " + yearModel + " " + make + " is now " + speed + ".");
+        }
```

Comment:
You could leave out the speed assignment and just put vehicle.getSpeed() in the println statement. That would save an extra step and the creation of a variable.